packages/components/src/shims-artplayer.d.ts

```diff
@@ -0,0 +1,173 @@
+declare module "artplayer/dist/artplayer.js" {
+  import Artplayer from "artplayer";
+  export = Artplayer;
+}
+
+declare module "artplayer-plugin-danmuku" {
+  import type Artplayer from "artplayer";
+
+  export = artplayerPluginDanmuku;
+  export as namespace artplayerPluginDanmuku;
+
+  export type Danmu = {
+    /**
+     * 弹幕文本
+     */
+    text: string;
+
+    /**
+     * 弹幕发送模式，0为滚动，1为静止
+     */
+    mode?: 0 | 1;
+
+    /**
+     * 弹幕颜色
+     */
+    color?: string;
+
+    /**
+     * 弹幕出现的时间，单位为秒
+     */
+    time?: number;
+
+    /**
+     * 弹幕是否有描边
+     */
+    border?: boolean;
```

Comment:
Default value?

Reply:
https://artplayer.org/document/plugin/danmuku.html#%E9%80%89%E9%A1%B9

Reply:
So why should time be optional